packages/@aws-cdk/toolkit/test/_helpers/index.ts

```diff
@@ -3,5 +3,5 @@ import * as path from 'path';
 export * from './test-io-host';
 
 export function fixture(name: string, app = 'app.js'): string {
-  return path.normalize(path.join(__dirname, '..', 'fixtures', name, app));
+  return path.normalize(path.join(__dirname, '..', '_fixtures', name, app));
```

Comment:
assume that you will need this in the origin branch and then this diff will go away